pkg/manifest/os.go

```diff
@@ -726,6 +727,17 @@ func (p *OS) serialize() osbuild.Pipeline {
 		}))
 	}
 
+	if p.OpenSCAPConfig != nil {
+		// nest this within the openscap config, since
+		// it doesn't make sense to have tailoring configs
+		// without openscap config
+		if p.OpenSCAPAutotailorConfig != nil {
```

Comment:
You can use `p.OpenSCAPConfig.Tailoring` instead

Reply:
Also, shouldn't you call `GetAutotailorFile()` here and append dirs and files to the list?

Reply:
The OpenSCAP autotailoring config object isn't in the OpenSCAPConfig though... the only thing that is saved there is the `filepath` to the tailoring file. So basically the two stages depend on each other here, but not completely.

Reply:
fair point... It is fine as is...